Assets/MixedRealityToolkit.Tests/PlayModeTests/Core/ChangeActiveProfileTests.cs

```diff
@@ -0,0 +1,157 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if !WINDOWS_UWP
+// When the .NET scripting backend is enabled and C# projects are built
+// The assembly that this file is part of is still built for the player,
+// even though the assembly itself is marked as a test assembly (this is not
+// expected because test assemblies should not be included in player builds).
+// Because the .NET backend is deprecated in 2018 and removed in 2019 and this
+// issue will likely persist for 2018, this issue is worked around by wrapping all
+// play mode tests in this check.
+
+using Microsoft.MixedReality.Toolkit.Boundary;
+using NUnit.Framework;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    // Tests to verify that changing the active profile at runtime results in
+    // the MRTK being correctly reconfigured.
+    public class ChangeActiveProfileTests
```

Comment:
Suggestion: Maybe add another test that loads HandInteractionExampleScene and then switchs the active profile from HL2 to DefaultMRTK or something? Should be simple to write up and will cover over many components

Reply:
good idea. i had thought of a test that ensures unity ui behavior (which I just confirmed in play mode)

Reply:
will add the test in a followup PR when i address issues with unity ui after profile change